homeassistant/components/nut/const.py

```diff
@@ -39,6 +39,8 @@
     "BOOST": "Boosting Voltage",
     "FSD": "Forced Shutdown",
     "ALARM": "Alarm",
+    "HE": "ECO Mode",
+    "TEST": "Battery Testing",
```

Comment:
At some point these should be converted to translations, but there is no harm in adding more and converting them later